.github/workflows/install_dependencies/action.yml

```diff
@@ -34,3 +34,4 @@ runs:
         pip install -r core/requirements_test.txt --use-pep517
         pip install -e core/ --use-pep517
         pip install $(python test_dependencies.py) --use-pep517
+        pip install jaraco.functools
```

Comment:
You probably want to install this in the Dockerfile though?

Reply:
I'm honestly not sure how that all works yet. Where does this line need to go?

Reply:
In Dockerfile

Reply:
I removed it here. We can add it to the Dockerfile separately.

What did you want to do with that package BTW? The docs of `jaraco.functools` give me a 404.